src/networks.py

```diff
@@ -7,8 +7,8 @@
         'genesis_hash': '00000ffd590b1485b3caadc19b22e6379c733355108f107a430458cdf3407ab6' #Darkcoin src/chainparams.cpp L62 
     },
     'Darkcoin_testnet': {
-        'pubkey_address': 139, #Darkcoin src/chainparams.cpp L137
+        'pubkey_address': 140, #Darkcoin src/chainparams.cpp L137
```

Comment:
nit: Dash, L224
note: might worth to update all other references too (in some another PR maybe)